nautobot/extras/templates/extras/configcontext_edit.html

```diff
@@ -26,9 +26,9 @@
             {% render_field form.tenant_groups %}
             {% render_field form.tenants %}
             {% render_field form.device_redundancy_groups %}
-            {% render_field form.tags %}
         </div>
     </div>
+    {% include 'inc/extras_features_edit_form_fields.html' %}
     <div class="panel panel-default">
```

Comment:
I think this change is undesirable; again this is not "tags for this config context" it's "this config context applies to devices/vms that have the specified tag(s)". As such, grouping the `tags` field under the `Assignment` panel is correct.